ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackToastView.swift

```diff
@@ -75,6 +75,28 @@ enum AIChatFeedbackToastType: Equatable {
   }
 }
 
+enum AIChatFeedbackToastPrivacyWarning: Equatable {
+  case none
+  case liked(turnIndex: Int, turnId: String)
+  case disliked(turnIndex: Int, turnId: String)
+
+  static func == (
+    lhs: AIChatFeedbackToastPrivacyWarning,
+    rhs: AIChatFeedbackToastPrivacyWarning
+  ) -> Bool {
+    switch (lhs, rhs) {
+    case (.none, .none):
+      return true
+    case (.liked(let turnIndexA, let turnIdA), .liked(let turnIndexB, let turnIdB)):
+      return turnIndexA == turnIndexB && turnIdA == turnIdB
+    case (.disliked(let turnIndexA, let turnIdA), .disliked(let turnIndexB, let turnIdB)):
+      return turnIndexA == turnIndexB && turnIdA == turnIdB
+    default:
+      return false
+    }
+  }
+}
+
```

Comment:
Doesnt need to be an enum, just make it bundle the data needed for the API call:

```swift
struct AIChatFeedbackInfo: Hashable {
  var isLiked: Bool
  var turnId: String
}
```

Reply:
Fixed.